app/_locales/en/messages.json

```diff
@@ -1455,7 +1455,7 @@
     "message": "WARNING: Never disclose your backup phrase. Anyone with this phrase can take your Ether forever."
   },
   "secretPhrase": {
-    "message": "Enter your secret twelve word phrase here to restore your vault."
+    "message": "Enter your secret phrase here to restore your vault."
```

Comment:
With this change (and `importYourExisting`) we'll want to make sure our other language files get updated to reflect the difference as well. I still don't know what's the best practice here, if we can just resubmit this file for translation of the keys that have been updated or what.

Reply:
The best approach we have is to remove the translations and hope somebody re-adds them. Not a great process, but, hopefully we'll have a better one soon for some locales at least.

Reply:
As discussed on sync, taking the approach @Gudahtt mentioned for now, translations updated